web/themes/custom/cybersecurity/src/js/menu.js

```diff
@@ -1,17 +1,17 @@
-// Add class open to menu-mob on click of burger nav icon
-$(function () {
+(function ($) {
+  // Add class open to menu-mob on click of burger nav icon
   $(".burger-nav").on("click", function() {
     if (!$(".menu-mob").hasClass("open")) {
```

Comment:
This works but also check out: https://api.jquery.com/toggleClass/

Reply:
I will keep a note of this and update it after we get the site up to scratch for this show and tell at the end of the week.